test/plugins/individual-collateral/morpho-aave/MorphoAAVENonFiatCollateral.test.ts

```diff
@@ -0,0 +1,261 @@
+import { networkConfig } from '#/common/configuration'
+import { bn, fp } from '#/common/numbers'
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers'
+import { MockV3Aggregator } from '@typechain/MockV3Aggregator'
+import { TestICollateral } from '@typechain/TestICollateral'
+import {
+  ERC20Mock,
+  MockV3Aggregator__factory,
+  MorphoNonFiatCollateral,
+  MorphoNonFiatCollateral__factory,
+} from '@typechain/index'
+import { expect } from 'chai'
+import { BigNumber, BigNumberish } from 'ethers'
+import { parseEther, parseUnits } from 'ethers/lib/utils'
+import { ethers } from 'hardhat'
+import collateralTests from '../collateralTests'
+import { getResetFork } from '../helpers'
+import { CollateralOpts } from '../pluginTestTypes'
+import {
+  DEFAULT_THRESHOLD,
+  DELAY_UNTIL_DEFAULT,
+  FORK_BLOCK,
+  ORACLE_ERROR,
+  ORACLE_TIMEOUT,
+  PRICE_TIMEOUT,
+} from './constants'
+import { MorphoAaveCollateralFixtureContext, mintCollateralTo } from './mintCollateralTo'
+
+interface MAFiatCollateralOpts extends CollateralOpts {
+  underlyingToken?: string
+  poolToken?: string
+  defaultPrice?: BigNumberish
+  defaultRefPerTok?: BigNumberish
+
+  refPerTokChainlinkFeed?: string
```

Comment:
all throughout this file I think the naming is refPerTok when it should be targetPerRef